app/views/infos/how_to_use.haml

```diff
@@ -7,7 +7,10 @@
 
 .left-navi-section.about-section
   = render :layout => "layouts/mercury_editable_page", :locals => { :content_type => "how_to_use_page_content" } do
-    - content = how_to_use_content.html_safe
+    - if @community_customization && ! @community_customization.how_to_use_page_content.nil?
```

Comment:
do you mean there shouldn't be space after !
yea, it might be bit uncommon. I've maybe done that few times as I myself tend to miss the ! if it's connected to the word. But better to have uniform convention and me learning to look right than changing it because of that.. :)